### PR TITLE
fix 2fa backup codes typo

### PR DESCRIPTION
**Pull request by ChristophWurst**
@nickvergessen @rullzer

### CODE REVIEW

**Comment by rullzer** [comment]
LGTM

File: apps/twofactor_backupcodes/js/settingsview.js [hunk 1]
```diff
@@ -109,12 +109,12 @@
 		},
 		_onPrintBackupCodes: function () {
 			var url = this._getDownloadDataHref();
-			window.open(url, 'Nextcloud backpu codes');
+			window.open(url, 'Nextcloud backup codes');
```
Comment (nickvergessen) [inline apps/twofactor_backupcodes/js/settingsview.js]: btw should this be translatable?

Comment (nickvergessen) [inline apps/twofactor_backupcodes/js/settingsview.js]: Same for all other strings in here

Comment (ChristophWurst) [inline apps/twofactor_backupcodes/js/settingsview.js]: good catch. Fixed that (found only one other string that was not yet translated)

**Comment by ChristophWurst** [comment]
btw, will the transifex export/import automatically find this new app or do we have to add this somewhere? cc @MorrisJobke

**Comment by nickvergessen** [comment]
:+1:

**Comment by MorrisJobke** [comment]
> btw, will the transifex export/import automatically find this new app or do we have to add this somewhere? cc @MorrisJobke

I will add it

**Comment by MorrisJobke** [comment]
> I will add it

#1286

**Pull request merged**